xla/service/cpu/tests/onednn_matmul_test.cc

```diff
@@ -1574,6 +1584,103 @@ TEST_F(MatmulTest, BroadcastedAddAfterFusion) {
   )");
 }
 
+// Test MM + BiasAdd + Add fusion : F32
```

Comment:
Nit: MM -> MatMul
```suggestion
// Test MatMul + BiasAdd + Add fusion : F32
```
Same for other places.